src/osd/PG.cc

```diff
@@ -3671,9 +3671,17 @@ int PG::_prepare_write_info(CephContext* cct,
   if (logger)
     logger->inc(l_osd_pg_info);
 
-  // try to do info efficiently?
-  if (!dirty_big_info && try_fast_info &&
+  if (info.last_update < last_written_info.last_update) {
+    // last_update rewound, fast_info probably no longer valid.
+    // compulsively  update fast_info and info togather.
+    pg_fast_info_t fast;
+    fast.last_update = info.last_update;
+    encode(fast, (*km)[fastinfo_key]);
```

Comment:
Seems you are trying to encode an empty fast_info with only the last_update field filled here...
Could you call  `fast.populate_from(info)` to get the fast_info correctly initialized?

Reply:
Thanks for review, seems that we filled the last_update field of fast_info is enough, because here we want to disable the fast_info, i means set last_update to older value equals delete it.